### DISCUSSION TITLE
Is Fudforum going forward?

### DISCUSSION THREAD

**Discussion by hair-transplant** [fudforum/FUDforum#40] [General]
As a long-time user with 4 big forums I really have mixed feelings now. Some recent code changes included a bug, which caused one of my forums to stop being usable. Unfortunately the same bug was in the fudforum.org website, so I could not post it there.
Now the Homepage stop working at all.
Is the project abandoned ?
Or just sign of work to repair ?

**Comment by jethro-swan** [comment]
I have essentially the same questions. What's happening?

**Comment by naudefj** [comment]
It better be moving forward  - I have big forums relying on it.
I will continue to work on the code, even if it's only for my own sites, but we need to get more people involved.

**Reply by jethro-swan** [reply]
@naudefj  How can I contact you?

**Comment by naudefj** [comment]
I'll send you an e-mail.

**Reply by jethro-swan** [reply]
An email I sent to one variant of the address bounced. I tried a different variant which didn't bounce; can you let me know whether you received that?

**Comment by hair-transplant** [comment]
Not everyone is a coder. Maybe give some money ?

**Comment by hair-transplant** [comment]
to hire someone to help you.
You seem to have found the bug which  crashed also my forum.
I think Fudforum just needs to get updated to PHP changes, I dont miss any function

**Comment by hair-transplant** [comment]
Hallo!

I received only this mail below
(for hairtransplant account)
 
Kind regards 
Ludger Mentrup
Rathenaustr. 4
63067 Offenbach
Tel. 00496917537900  (AB)
0152 - 28937675
 
Sender: John Waters
Send Time: 2023-11-20 21:55
Receiver: fudforum/FUDforum
cc: hair-transplant; Author
Subject: Re: [fudforum/FUDforum] Is Fudforum going forward? (Discussion #40)
 
An email I sent to one variant of the address bounced. I tried a different variant which didn't bounce; can you let me know whether you received that?

**Reply by jethro-swan** [reply]
Sorry @hair-transplant, that message was intended for @naudefj whom I'm trying to contact.
